front_laser:
  x: 0
  y: 0
  theta: 0
  topic: /front_scan 
  is_real_time: False
  need_filter: False

back_laser:
  x: -0.6
  y: -0.03
  theta: 3.141592653589793
  topic: /back_scan
  is_real_time: False
  need_filter: False

